extensions/pcap/CapturePacket.h

```diff
@@ -16,8 +16,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#ifndef __INVOKE_HTTP_H__
-#define __INVOKE_HTTP_H__
+#ifndef EXTENSIONS_CAPTUREPACKET_H
+#define EXTENSIONS_CAPTUREPACKET_H
```

Comment:
Nice catch, thanks for that!

Please check the style in standard processors. The closing #endif at the end of the file should have a comment to indicate which header guard it closes.

Reply:
Sure, I'll change that.  That is *already* the case however, which I guess given the header guard thing isn't surprising.  I left it the way it was.